File: 240729/숫자 직사각형/rectangle-of-numbers.c
#include <stdio.h>

int main() {
    // input: n m (행 n, 열m)
    int n,m;
    scanf("%d %d", &n, &m);

    int arr[n][m];
    int num = 1; // 1로 초기화
    
    for (int i=0; i<n; i++){
        for (int j=0; j<m; j++) {
            arr[i][j] = num;
            num += 1;
        }
    }

    for (int i=0; i<n; i++){
        for (int j=0; j<m; j++) {
            printf("%d ", arr[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: 240731/숫자 직사각형/rectangle-of-numbers.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        // 배열 초기화
        int n = sc.nextInt();
        int m = sc.nextInt();
        int[][] arr = new int[n][m];
        
        // 숫자 넣어서 짜기
        int num = 1;
        for (int i=0; i<n; i++) {
            for (int j=0; j<m; j++) {
                arr[i][j] = num;
                num++;
                System.out.print(arr[i][j] + " ");
            }
            System.out.print("\n");
        }
    }
}